clickhouse/base/socket.cpp

```diff
@@ -182,6 +182,7 @@ NetworkAddress::NetworkAddress(const std::string& host, const std::string& port)
     hints.ai_family = PF_UNSPEC;
     hints.ai_socktype = SOCK_STREAM;
 
+#if defined(_unix_) 
```

Comment:
Please add some context to the fix in comments, like adding `AI_ADDRCONFIG` on Windows causes `getaddrinfo` to fail with `WSAHOST_NOT_FOUND` on host addr "".